src/commands/txCommands/util/index.js

```diff
@@ -382,6 +369,47 @@ const resourcesComplete$ = resourceCompletion$
 	.filter(item => Object.keys(item[1]).length === 0)
 	.map(([resource]) => resource);
 
+// Helper to grab all local trns to be merged to a tx resource (e.g. default English messages)
+const updateAllMessages$ = (resource, project) =>
+	localTrnsMerged$
+		.flatMap(poContent =>
+			updateResource$(
+				resource,
+				poContent,
+				project,
+			).do(
+				() => console.log(`update ${project} - ${resource} success`),
+				(error) => console.log(`update ${project} - ${resource} FAIL!`, error))
+		) // update resource
```

Comment:
this is an odd place for this inline comment, and I think it's clear from the `updateResource$` function name.